Softeq.XToolkit.WhiteLabel/Navigation/NavigateHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Reflection;
+using Softeq.XToolkit.WhiteLabel.Extensions;
+using Softeq.XToolkit.WhiteLabel.Mvvm;
+
+namespace Softeq.XToolkit.WhiteLabel.Navigation
+{
+    public class NavigateHelper<TViewModel> where TViewModel : IViewModelBase

```

Comment:
Maybe rename to `Builder` or something else, because of Helper associated with static class.